robot/Cumulus/tests/browser/donations_payments/create_donation_and_payment.robot

```diff
@@ -36,6 +36,9 @@ Create Donation from a Contact
     Go To Page                           Details
     ...                                  Opportunity
     ...                                  object_id=${data}[contact_opportunity][Id]
```

Comment:
u can remove this as u included go to page in other keyword

Reply:
I need this because The above keyword is in the opportunities page object and I cannot access it without navigating to the opportunities page.